.github/workflows/fps-submissions-checks.yml

```diff
@@ -46,8 +51,9 @@ jobs:
       - name: Write Comment on PR
         uses: mshick/add-pr-comment@v2
         with:   
-          message-path: 
-            "message.txt" 
+          message-path: "message.txt" 
+          allow-repeats: true
+          refresh-message-position: true
```

Comment:
It looks like `refresh-message-position` only has an effect if `allow-repeats` is false: https://github.com/mshick/add-pr-comment/blob/387ece43e304d694bf3f6f9bdc6188b1d7f1d90f/src/main.ts#L53-L79

So I would just use the `refresh-message-position` option, and remove the `allow-repeats`.

Reply:
Done